.vscode/extensions.json

```diff
@@ -1,7 +1,6 @@
 {
   "recommendations": [
     "esbenp.prettier-vscode",
-    "editorconfig.editorconfig",
```

Comment:
Intended?

Reply:
Not sure why this was resolved. This change is the one currently blocking the merge 😄 
@fbinotto's approval is enough for the module to be merged, but if it's anything outside a module (like in this case a settings file), this is when a maintainer must approve. Please just undo this change and we can get the PR in @bobmclane999  😉